tasmota/settings.ino

```diff
@@ -1243,11 +1243,11 @@ void SettingsDelta(void)
       Settings.ex_serial_config = TS_SERIAL_8N1;
     }
     if (Settings.version < 0x07010204) {
-    //   if (Settings.flag3.ex_cors_enabled == 1) {
-    //     strlcpy(Settings.ex_cors_domain, CORS_ENABLED_ALL, sizeof(Settings.ex_cors_domain));
-    //   } else {
-    //     Settings.ex_cors_domain[0] = 0;
-    //   }
+       if (Settings.flag3.mqtt_buttons == 1) {
```

Comment:
I thought was better remove it completely...

Reply:
Yes it requires a quirky mind but users coming from versions below 7.1.2.4 are now still able to migrate their previous setting.

Reply:
Understood. Thanks for the explaination.